docs/certificatessigningrequest-metrics.md

```diff
@@ -6,3 +6,4 @@
 | kube_certificatesigningrequest_condition | Gauge | `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt; <br> `condition`=&lt;approved\|denied&gt; | STABLE |
 | kube_certificatesigningrequest_labels | Gauge | `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt;| STABLE |
 | kube_certificatesigningrequest_cert_length | Gauge | `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt;| STABLE |
+| kube_certificatesigningrequest_annotations | Guage | `annotation_CSR_ANNOTATION`=&lt;CSR_ANNOTATION&gt; <br> `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt; | EXPERIMENTAL |
```

Comment:
I see  this typo everywhere.
```suggestion
| kube_certificatesigningrequest_annotations | Gauge | `annotation_CSR_ANNOTATION`=&lt;CSR_ANNOTATION&gt; <br> `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt; | EXPERIMENTAL |
```

Reply:
What's wrong with that? I took this type from here https://github.com/kubernetes/kube-state-metrics/blame/master/docs/namespace-metrics.md#L7

Reply:
I said typo, not type :). Guage needs to be Gauge.

Reply:
Once the changes are made, please squash the commits.

Reply:
I apologize. Done.

Reply:
No worries at all!